dev-resources/Documents/elrr_setup.txt

```diff
@@ -0,0 +1,233 @@
+elrr
```

Comment:
So this is a little more like the docs you would put in here. However it should be in .md, like the readme, and should only pertain to THIS project, with linked references to the others.